src/Xamarin.Android.Tools.AndroidSdk/Sdks/AndroidSdkUnix.cs

```diff
@@ -240,5 +303,37 @@ private XDocument GetUnixConfigFile ()
 			return doc;
 		}
 
+		void FixOwnership (List<string> paths)
+		{
+			if (!need_chown || paths == null || paths.Count == 0)
+				return;
+
+			var stdout = new StringWriter ();
+			var stderr = new StringWriter ();
+			var args = new List <string> {
+				QuoteString (sudo_user)
```

Comment:
I don't see the need for `QuoteString()`. Why not "quote" the string instead of escaping spaces, e.g.:

```csharp
var args = new List<string> {
    $"\"{sudo_user}\"",
};
```

I feel safer about *actually* quoting strings and not escaping certain characters.

Reply:
I made the change in `QuoteString`, to keep quoting in a single place should we need to change it.